config/barclamps.yml

```diff
@@ -1,33 +1,5 @@
-crowbar:
-  - cisco-ucs
-  - crowbar
-  - deployer
-  - dns
-  - ipmi
-  - logging
-  - network
-  - nfs_client
-  - ntp
-  - provisioner
-  - suse-manager-client
-  - updater
-
-openstack:
-  - ceilometer
-  - ceph
-  - cinder
-  - database
-  - glance
-  - heat
-  - hyperv-compute
-  - keystone
-  - manila
-  - neutron
-  - nova
-  - nova_dashboard
-  - openstack
-  - pacemaker
-  - rabbitmq
-  - swift
-  - tempest
-  - trove
+- crowbar-core
+- crowbar-openstack
+- crowbar-ceph
+- corbwar-hyperv
```

Comment:
This is already fixed in an followup pull request. This pull request missed a fixing commit.